content/en/blog/_posts/2020-12-10-Kubernetes Volume Snapshot Moves to GA.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: blog
+title: 'Kubernetes 1.20: Kubernetes Volume Snapshot Moves to GA'
+date: 2020-12-10
+slug: kubernetes-release-1.20-Kubernetes-Volume-Snapshot-Moves-to-GA
```

Comment:
How about:
```suggestion
slug: kubernetes-1.20-volume-snapshot-moves-to-ga
```
?

Reply:
This makes total sense! I thought it was repetitive too :)